hyp3_gamma/water_mask.py

```diff
@@ -43,6 +45,37 @@ def split_geometry_on_antimeridian(geometry: dict):
     return json.loads(geojson_str)['features'][0]['geometry']
 
 
+def poly_from_box(box):
+    '''
+    create a polygon with box
+    Args:
+        box: [min_lon, min_lat, max_lon, max_lat]
+    Returns:
+        polygon
+    '''
+    p1 = geometry.Point(box[0], box[1])
+    p2 = geometry.Point(box[2], box[1])
+    p3 = geometry.Point(box[2], box[3])
+    p4 = geometry.Point(box[0], box[3])
+    return geometry.Polygon([p1, p2, p3, p4, p1])
```

Comment:
An additional point was at the end of this list, and I think it's a bit clearer like this.
```suggestion
    min_lon = geometry.Point(box[0], box[1])
    min_lat = geometry.Point(box[2], box[1])
    max_lon = geometry.Point(box[2], box[3])
    max_lat = geometry.Point(box[0], box[3])
    return geometry.Polygon([min_lon, min_lat, max_lon, max_lat])
```

Reply:
Other than that, everything looks good. I verified for correctness of the masks in a local environment.